packages/netlify-cms-core/src/components/Collection/Entries/Entries.js

```diff
@@ -45,8 +46,9 @@ const Entries = ({
           viewStyle={viewStyle}
           cursor={cursor}
           handleCursorActions={handleCursorActions}
+          page={page}
         />
-        {isFetching && page !== undefined ? (
+        {isFetching && page !== undefined && entries.size > 0 ? (
```

Comment:
This prevents flickering `Loading entries` message while fetching the first pages.